app/Http/Controllers/Admin/RouteController.php

```diff
@@ -40,8 +43,32 @@ public function create()
      */
     public function store(RouteRequest $request)
     {
-        $route = new Route($request->all());
-        $route->save();
+        $allRequest = $request->all();
+        $routeRequest = $request->only(['name','distance','frequency','frequency_peak','start_time','end_time','type']);
```

Comment:
just get all request params, no need to limit. Because you already have `fillable` in model. If you don't understand `$fillable` pls research